test/selenium/lib/view/signed_out/view.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var SignedOutPageAccessor = require('./accessors.js');
+
+function SignedOutPageView(driver) {
+    this.driver = driver;
+    this.accessors = new SignedOutPageAccessor(this.driver);
+};
+
+SignedOutPageView.prototype = {
+    confirmPage: function() {
```

Comment:
I wanted to be sure I was on the correct page. If I understood your previous comment, and changing the code, removing this function and doing and assert.equal(present, true) would be enough...will try that..